### ISSUE TITLE
Reverb FX in Sample Editor

### ISSUE THREAD

**Issue by frognerboy** [polyend/TrackerIssues#1169]
Adding a reverb in sample editor would be awesome. You could then easily liven up a dry snare and not "spend" up any other real-time FX.

**Comment by Djeroek** [comment]
Yes please! Was about to make a similar suggestion and found this thread. The more ways to offline manipulate sampled content the better imo.
